renku/cli/_providers/zenodo.py

```diff
@@ -149,3 +156,24 @@ def get_record(self, uri):
             raise LookupError('record not found')
 
         return ZenodoRecord(**response.json(), zenodo=self)
+
+
+class ZenodoMetadata:
+    """Zenodo metadata."""
+
+    @staticmethod
+    def from_renku_dataset(dataset):
+        """Serialize renku dataset to Zenodo dataset."""
+        # TODO: #512, #511
+        return {
```

Comment:
we should aim to use json-ld contexts for this rather than hand-written json. Should hopefully be easier with #527.

Reply:
Yes, let's get it merged. :smile: